src/tools/debugger/session/Breakpoints.java

```diff
@@ -54,13 +59,16 @@
   /** Manually managed by us, instead of Truffle. */
   private final Map<FullSourceCoordinate, BreakpointEnabling<ChannelOppositeBreakpoint>> channelOppositeBreakpoint;
 
+  private Stepping stepping;
+
   public Breakpoints(final Debugger debugger, final WebDebugger webDebugger) {
     this.truffleBreakpoints           = new HashMap<>();
     this.receiverBreakpoints          = new HashMap<>();
     this.promiseResolverBreakpoints   = new HashMap<>();
     this.promiseResolutionBreakpoints = new HashMap<>();
     this.channelOppositeBreakpoint    = new HashMap<>();
     this.debuggerSession = debugger.startSession(webDebugger);
+    this.stepping = null;
```

Comment:
I don't think it is possible to have global stepping field. I don't see how that can be meaningful.